qa/tasks/ceph.py

```diff
@@ -69,6 +69,35 @@ def generate_caps(type_):
         yield capability
 
 
+@contextlib.contextmanager
```

Comment:
@shyukri could you drop this function as a whole? as mimic does not have `ceph_crash()` by then.